TreeDB/docs/spec/storage-format.md

```
@@ -494,8 +494,9 @@ Insert/update rows must have `Deleted=false` and exactly one value per declared
 `Deleted=true` and zero column values. For layouts with `typed_column_part`
 owners, a `TCPA` row asset is still published for row IDs/tombstones and any
 row-owned fields; the matching `tcs1_typed_column_part` for the same non-delete
-generation contains authoritative scalar, fixed-dimension `float32_vector`, and
-non-null variable-width `uint32_list` typed-column values keyed by row index.
+generation contains authoritative scalar, fixed-dimension `float32_vector`,
+#1930 dense numeric vector, and non-null variable-width `uint32_list`
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)